lib/install.js

```diff
@@ -46,6 +46,11 @@ function install(opts, cb) {
     opts.drivers = defaultConfig.drivers;
   }
 
+  if (process.platform !== 'win32') {
+    delete opts.drivers.ie;
```

Comment:
no problem, thanks for your speedy response!  I'm pointing at your latest again in my work project.  👍 

Cheers!